packages/account/src/BaseAccount.ts

```diff
@@ -18,12 +18,21 @@ export abstract class SmartAccount implements ISmartAccount {
   paymaster!: IPaymaster
   initCode: string = '0x'
   proxy!: SmartAccount_v100
+  owner!: string
   provider!: JsonRpcProvider
   entryPoint!: EntryPoint_v100
   chainId!: ChainId
   signer!: Signer
+  smartAccountConfig: SmartAccountConfig
 
-  constructor(readonly smartAccountConfig: SmartAccountConfig) { }
+  constructor(_smartAccountConfig: SmartAccountConfig) {
+    this.smartAccountConfig = _smartAccountConfig
+   }
+
+
+  setEntrypointAddress(entryPointAddress: string){
```

Comment:
Use proper camel case in function names.
It should be `setEntryPointAddress`

Reply:
resolved